src/main/java/org/folio/edge/api/utils/util/PropertiesUtil.java

```diff
@@ -0,0 +1,50 @@
+package org.folio.edge.api.utils.util;
+
+import static java.lang.String.format;
```

Comment:
we're using log4j2, right?  So let's use the built-in variable substitution instead of String.format

Reply:
updated